chore(deps): bump @types/lodash from 4.14.146 to 4.14.149 in stark-core, stark-rbac and stark-ui

SuperITMan opened the pull request:
Update package-lock.json files in:
  - root folder
  - stark-build
  - stark-testing

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[X] Other... Please describe: Update dependencies
```

## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

The dependencies "@types/lodash" and "@angular/*" are outdated in "package-lock.json" files in the Stark packages and in the root folder.

Issue Number: N/A


## What is the new behavior?

Updated package-lock.json files in the Stark packages and in the root folder.

## Does this PR introduce a breaking change?
```
[ ] Yes
[X] No
```

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

christophercr commented on package-lock.json:
```diff
@@ -94,9 +94,9 @@
     },
     "@angular/animations": {
-      "version": "7.2.15",
-      "resolved": "https://registry.npmjs.org/@angular/animations/-/animations-7.2.15.tgz",
```
This `package-lock.json` file doesn't need to be updated since there is no `@types/lodash` in here... could you remove this file from this PR?

christophercr commented on packages/stark-testing/package-lock.json:
```diff
@@ -5,95 +5,95 @@
   "requires": true,
   "dependencies": {
     "@babel/code-frame": {
-      "version": "7.5.5",
```
This `package-lock.json` file doesn't need to be updated since there is no `@types/lodash` in here... could you remove this file from this PR?

SuperITMan commented:
Actually, my commit message is incorrect according to what I changed. I update my commit with the correct message and I added the missing stark-build 😊

SuperITMan commented:
@christophercr PR updated
